CMakeLists.txt

```diff
@@ -143,6 +143,14 @@ else()
 	set(DOLPHIN_VERSION_PATCH ${DOLPHIN_WC_REVISION})
 endif()
 
+# If Dolphin is not built from a Git repository, default the version info to
+# reasonable values.
+if(NOT DOLPHIN_WC_REVISION)
+	set(DOLPHIN_WC_DESCRIBE "${DOLPHIN_VERSION_MAJOR}.${DOLPHIN_VERSION_MINOR}")
+	set(DOLPHIN_WC_REVISION "${DOLPHIN_WC_DESCRIBE} (no further info)")
```

Comment:
There will also be some oddities with how Movie.cpp writes the revision to DTM files, because it expects that scm_rev_git_str only contains hexadecimal digits. I can make a PR to improve Movie.cpp's handling for that later.